log.go

```diff
@@ -116,9 +127,10 @@ func (log *defaultLogger) WithContext(ctx context.Context) *rlog.Entry {
 // CreateDefaultLogger return a new instance of SFLogger with default config
 func CreateDefaultLogger() SFLogger {
 	var rLogger = rlog.New()
-	var formatter = rlog.TextFormatter{CallerPrettyfier: SFCallerPrettyfier}
+	var formatter = new(sfTextFormatter)
```

Comment:
Maybe we can leverage some more golang syntax here?

```
rLogger := rlog.New()
formatter := &sfTextFormatter{
    CallerPrettyfier: SFCallerPrettyfier,
}
```
WDYT?

Reply:
It doesn't work. I get the error `unknown field CallerPrettyfier in struct literal of type sfTextFormatter`.